repose-aggregator/components/services/atom-feed-service/atom-feed-service-impl/src/main/scala/org/openrepose/nodeservice/atomfeed/impl/auth/OpenStackIdentityV2AuthenticatedRequestFactory.scala

```diff
@@ -44,7 +45,7 @@ object OpenStackIdentityV2AuthenticatedRequestFactory {
 /**
   * Fetches a token from the OpenStack Identity service, if necessary, then adds the token to the request.
   */
-class OpenStackIdentityV2AuthenticatedRequestFactory(configuration: OpenStackIdentityV2AuthenticationType)
+class OpenStackIdentityV2AuthenticatedRequestFactory @Inject()(configuration: OpenStackIdentityV2AuthenticationType)
```

Comment:
this was supposed to also use the connection pool, with the same configured pool as it's feed